config/routes.rb

```diff
@@ -18,14 +17,16 @@
   
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
-      resources :test_covids
+    resources :test_covids
 	  resources :user_history_details
 	  resources :user_histories
 	  resources :area_histories
 	  resources :areas
 	  resources :symptoms
-	  resources :insititutions
+	  resources :institutions
 	  devise_for :users
+
+    post 'institutions/search', :to => 'institutions#search'
```

Comment:
POST is supposed to be used to mutate data in the DB.
WDTY about using `GET` here?

Reply:
Sorry my bad, you are right, I must change it, thanks for your comments